### ISSUE TITLE
Incorrect default

### ISSUE THREAD

**Issue by AlanOlrog** [MicrosoftDocs/VBA-Docs#88]
My testing indicates that (unlike the Weekday function) the VBA WeekdayName function does not use vbSunday for the default firstdayofweek argument, as stated, but uses vbUseSystem.
To illustrate, if the system "First day of week" is set to Tuesday, WeekDayName(vbWednesday,,vbUseSystem) returns "Friday", as expected, but WeekDayName(vbWednesday) also returns "Friday". In contrast, WeekDayName(vbWednesday,,vbSunday) returns "Wednesday", which shows that omitting the firstdayofweek argument causes vbUseSystem to be used.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 535483a0-c9c6-d475-7608-f8f9cf17f180
* Version Independent ID: e19f07ef-023a-fbd3-ac2d-afe6764d2462
* Content: [WeekdayName Function](https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/weekdayname-function)
* Content Source: [Language/Reference/User-Interface-Help/weekdayname-function.md](https://github.com/MicrosoftDocs/VBA-Docs/blob/master/Language/Reference/User-Interface-Help/weekdayname-function.md)
* Product: **office**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by lindalu-MSFT** [comment]
@AlanOlrog Have you tried Stack Overflow? You can ask the developer community questions about your VBA code directly. This question was asked/answered here:
https://stackoverflow.com/questions/41206775/excel-vba-if-weekday-of-file-created-date-matches-cell-value/41207153#41207153

This github issues forum is for bugs in our documentation. Hope this helps, closing issue.

**Comment by AlanOlrog** [comment]
Hello Linda, and thanks for your response - this was my first-ever GitHub post!
You've left me confused, though. Clearly, I wasn't asking a question: I reported an error in the documentation for the VBA WeekdayName function which, as you say, is what this GitHub forum is for. I'm aware of Stack Overflow, and your link confirms the error I reported. To summarise, the error I reported is that the WeekdayName function uses **vbUseSystem** - not vbSunday - as the default *firstdayofweek* argument.
